playbooks/common/openshift-etcd/config.yml

```diff
@@ -0,0 +1,106 @@
+---
+- name: Gather and set facts for etcd hosts
+  hosts: oo_etcd_hosts_to_config
+  roles:
+  - openshift_facts
+  tasks:
+  - openshift_facts:
+      role: common
+      local_facts:
+        hostname: "{{ openshift_hostname | default(None) }}"
+  - name: Check for etcd certificates
+    stat:
+      path: "{{ item }}"
+    with_items:
+    - "/etc/etcd/ca.crt"
+    - "/etc/etcd/client.crt"
+    - "/etc/etcd/client.key"
+    - "/etc/etcd/peer-ca.crt"
+    - "/etc/etcd/peer.crt"
+    - "/etc/etcd/peer.key"
+    register: stat_result
```

Comment:
Well, it's not that it "needs" to be a global, it just is a global because of how ansible handles register variables.

We do have an open discussion point on included playbooks and variables that we can discuss next week in the ansible arch meeting.

Until that's decided, we've just been using g_ for all global vars.